sdk/identity/Azure.Identity/src/IdentityClientOptions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.Pipeline;
+using Azure.Core.Pipeline.Policies;
+
+namespace Azure.Identity
+{    public class IdentityClientOptions : HttpClientOptions
+    {
+        private const string DefaultAuthorityHost = "https://login.microsoftonline.com/";
+        private readonly static TimeSpan DefaultRefreshBuffer = TimeSpan.FromMinutes(2);
+
+        public RetryPolicy RetryPolicy { get; set; }
+
+        public HttpPipelinePolicy LoggingPolicy { get; set; }
```

Comment:
@pakrym, why is logging policy not on the base class?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/6289

Reply:
To that end shouldn't RetryPolicy also be on the base?

Reply:
Nope, we don't know which kind of retry policy the client would want to use (fixed, exponential) so it has to be defined on the client specific options at the moment.

Reply:
But both derive from RetryPolicy.  The client specific options can specify what the default is in it's constructor, but should the client specific options totally restrict what types of retries policies should be allowed?  If a service by default uses a fixed policy should the user not be able to override this with a exponential policy and vice versa? Shouldn't user's also be able to specify their own RetryPolicy implementations?

Reply:
I 100% agree with you. This is one of the problems we'd like to resolve as part of https://github.com/Azure/azure-sdk-for-net/issues/6159.

The problem is to find a balance between being able to set policy specific options (exponential retry policy has a different set of options than fixed retry policy) in a strongly typed way and being able to override policies without artificial constraints.